plugin.py

```
@@ -927,6 +927,127 @@ def _db_stats() -> str:
+    """
+    Database cleanup operations using Django ORM.
+    Issue #556 - Quick database cleanup for development/debugging.
+
+    Args:
+        cleanup_type: One of "list", "series", "all_episodes", "all_series", "all_movies"
+        series_id: Required for cleanup_type="series"
+
+    Returns:
+        Formatted string with operation results
+    """
+    from django.db import transaction
```

Ruff (0.14.3)

Redefinition of unused `transaction` from line 40

Remove definition: `transaction`

(F811)